pwndbg/commands/got.py

```diff
@@ -12,44 +12,37 @@
 import pwndbg.enhance
 import pwndbg.file
 import pwndbg.which
-import pwndbg.wrappers
+import pwndbg.wrappers.checksec
+import pwndbg.wrappers.readelf
 
 from pwndbg.color import green
 from pwndbg.color import light_yellow
 
-
 parser = argparse.ArgumentParser(description='Show the state of the Global Offset Table')
 parser.add_argument('name_filter', help='Filter results by passed name.',
                     type=str, nargs='?', default='')
 
-
 @pwndbg.commands.ArgparsedCommand(parser)
 @pwndbg.commands.OnlyWhenRunning
 def got(name_filter=''):
-    local_path = pwndbg.file.get_file(pwndbg.proc.exe)
-    cs_out = pwndbg.wrappers.checksec("--file", local_path)
-
-    file_out = pwndbg.wrappers.file(local_path)
-    if "statically" in file_out:
-        return "Binary is statically linked."
-
-    readelf_out = pwndbg.wrappers.readelf("-r", local_path)
-
-    jmpslots = '\n'.join(filter(lambda l: _extract_jumps(l),
-                         readelf_out.splitlines()))
+    '''
+    Show the state of the Global Offset Table
```

Comment:
Not necessary as this is argparsed command. This is not used anywhere - except e.g. for an IDE - but lets not split description into two places as it might not be synchronized properly in the future.